packages/monaco-textmate/data/grammars/javascript.tmlanguage.json

```diff
@@ -0,0 +1,516 @@
+{
+  "comment": "JavaScript Syntax: version 2.0",
```

Comment:
grammars belong into the respective language extensions. I.e. JS & TS into our typescript extension.

Reply:
It is pretty specific to textmate though, what about languages without a proper Theia extension ?

Reply:
Textmate would become part of the language extension API. 
But for the many textmate grammars for which we don't have extensions, we could have a bulk extension including and registering them.

Reply:
There is a [package](https://github.com/NeekSandhu/rink-languages) that provides a lot of grammars already. Maybe this would do the trick ?